xml/mysql_banners.xml

```diff
@@ -903,8 +904,132 @@
+    <param pos="0" name="os.version" value="15.10"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:canonical:ubuntu_linux:13.10"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4})-MariaDB.+~vivid(?:-log)?$" flags="REG_ICASE">
+    <description>MariaDB Vivid</description>
+    <example service.version="10.0.24">5.5.5-10.0.24-MariaDB-1~vivid</example>
+    <example service.version="10.1.10">5.5.5-10.1.10-MariaDB-1~vivid-log</example>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="MariaDB"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MariaDB"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:mariadb:mariadb:{service.version}"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4})-MariaDB.+~vivid-wsrep(?:-log)?$" flags="REG_ICASE">
+    <description>MariaDB-wsrep</description>
+    <example service.version="10.0.24">5.5.5-10.0.24-MariaDB-1~vivid-wsrep</example>
+    <example service.version="10.2.9">5.5.5-10.2.9-MariaDB-1~vivid-wsrep-log</example>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="MariaDB"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MariaDB"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:mariadb:mariadb:{service.version}"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4})-MariaDB.+~eoan(?:-log)?$" flags="REG_ICASE">
+    <description>MariaDB on Ubuntu 19.10 (Eoan)</description>
+    <example service.version="10.3.22">5.5.5-10.3.22-MariaDB-1:10.3.22+maria~eoan</example>
+    <example service.version="10.4.12">5.5.5-10.4.12-MariaDB-1:10.4.12+maria~eoan-log</example>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="MariaDB"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MariaDB"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:mariadb:mariadb:{service.version}"/>
+    <param pos="0" name="os.vendor" value="Ubuntu"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="os.version" value="19.10"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:canonical:ubuntu_linux:13.10"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4})-MariaDB.+~cosmic(?:-log)?$" flags="REG_ICASE">
+    <description>MariaDB on Ubuntu 18.10 (Cosmic)</description>
+    <example service.version="10.3.17">5.5.5-10.3.17-MariaDB-1:10.3.17+maria~cosmic-log</example>
+    <example service.version="10.1.20">5.5.5-10.1.20-MariaDB-1:10.3.17+maria~cosmic-log</example>
```

Comment:
I'd remove the second example or replace it with one that exercises more of the regex